src/servers/src/opentsdb.rs

```diff
@@ -126,3 +130,38 @@ impl Server for OpentsdbServer {
         OPENTSDB_SERVER
     }
 }
+
+pub fn data_point_to_grpc_row_insert_requests(
+    data_points: &[DataPoint],
```

Comment:
Perhaps we should use `data_points: Vec<DataPoint>` to avoid cloning tags afterwards.

Reply:
Hmm, we'll still have to clone it since `data_point.tags()` returns a reference to a vec of tuple strings.

Reply:
We can  avoid to clone data from data_point: `mem::take(&mut data_point.tags)`